ci/check-style.sh

```diff
@@ -120,18 +81,30 @@ if ! find . \( "${ignore[@]}" \) -prune -o \
   problems="${problems} shellcheck"
 fi
 
-# Apply shfmt to format all shell scripts
-find . \( "${ignore[@]}" \) -prune -o \
-  -iname '*.sh' -print0 | xargs -0 shfmt -w -i 2
+# Apply several transformations that cannot be enforced by clang-format:
+#     - Replace any #include for grpc++/* with grpcpp/*. The paths with grpc++
+#       are obsoleted by the gRPC team, so we should not use them in our code.
+#     - Replace grpc::<BLAH> with grpc::StatusCode::<BLAH>, the aliases in the
+#       `grpc::` namespace do not exist inside google.
```

Comment:
Aside: We should probably justify this transformation more generally.

Reply:
ack